test/selenium-tests/index.test.js

```diff
@@ -0,0 +1,219 @@
+           assert.equal(issueEntry, 'issue');
+           assert.equal(employeeEntry, 'employee');
+        })
+
+        it('entry data on the right part should be taken over', async () => {
+           await driver.wait(until.elementLocated(page.buttonCreateEntrySelector()));
+           await driver.findElement(page.buttonCreateEntrySelector()).click();
+
+            await driver.wait(until.elementLocated(page.dateEntrySelector()));
+            await driver.wait(until.elementLocated(page.timeEntrySelector()));
+
+            await driver.findElement(page.dateEntrySelector()).click();
+            await driver.findElement(page.calendarDaySelector(1)).click();
+            const dateEntry = await driver.findElement(page.dateEntrySelector()).getAttribute('value');
+
+            await driver.findElement(page.timeOptionSelector(5)).click();
+            const timeEntry = await driver.findElement(page.timeEntrySelector()).getAttribute('value');
+
+            assert.notEqual(dateEntry, '');
+            assert.equal(timeEntry, '0.75');
+        })
+
+        it('should alert if machine, employee, date or time is not entered', async () => {
+            await driver.wait(until.elementLocated(page.buttonCreateEntrySelector()));
+            await driver.findElement(page.buttonCreateEntrySelector()).click();
+
+            await driver.wait(until.elementLocated(page.buttonConfirmSelector()));
+            await driver.findElement(page.buttonConfirmSelector()).click();
+
+            await driver.wait(until.alertIsPresent());
+            const alertMessage = await driver.switchTo().alert().getText();
+            await driver.switchTo().alert().accept();
+
+            assert.equal(alertMessage, 'At least machine, employee, date and time inputs must be made')
+        })
+
+        it('entry data should be taken over to table after confirmation', async () => {
+            await driver.wait(until.elementLocated(page.warrantyTableSelector()));
+            const amountOfWarrantyEntriesBeforeEntry = driver.findElements(page.warrantyTableEntrySelector());
+
+            await driver.wait(until.elementLocated(page.buttonCreateEntrySelector()));
+            await driver.findElement(page.buttonCreateEntrySelector()).click();
+
+            await driver.wait(until.elementLocated(page.machineEntrySelector()));
+            await driver.wait(until.elementLocated(page.employeeEntrySelector()));
+            await driver.wait(until.elementLocated(page.dateEntrySelector()));
+            await driver.wait(until.elementLocated(page.timeEntrySelector()));
+            await driver.wait(until.elementLocated(page.buttonConfirmSelector()));
+
+            await driver.findElement(page.machineEntrySelector()).sendKeys('machine');
+            await driver.findElement(page.employeeEntrySelector()).sendKeys('employee');
+            await driver.findElement(page.dateEntrySelector()).click();
+            await driver.findElement(page.calendarDaySelector(1)).click();
```

Comment:
Use a day in the middle of the date picker. That way you are sure that the picked day is within the current month.

Reply:
updated